bigtable/CMakeLists.txt

```diff
@@ -311,6 +317,13 @@ target_link_libraries(filters_integration_test
     gmock gRPC::grpc protobuf::libprotobuf)
 add_dependencies(tests-local filters_integration_test)
 
+# An integration test for bigtable::Mutations.
```

Comment:
I did not notice this earlier: this new program should be invoked from `run_integration_tests.sh` to make the tests part of the continuous integration run.  Please add that change.